packages/@react-spectrum/card/src/BaseLayout.tsx

```diff
@@ -80,6 +81,22 @@ export class BaseLayout<T> extends Layout<Node<T>> implements KeyboardDelegate {
       }
     }
 
+    // If there is an active editable card, make sure it remains in the DOM even when scrolled out of view
+    // so that we don't lose the card's focused child. This really only matters for virtualized card view
```

Comment:
Wonder how this affects screen readers. There'd be a gap in the rows that are "visible" potentially. We should test this.